mysql-test/suite/rocksdb/t/create_table.test

```diff
@@ -189,3 +190,41 @@ CREATE TABLE t1(c1 INT,c2 INT,c3 INT,c4 INT,c5 INT,c6 INT,c7 INT,c8 INT,c9 INT,
 SHOW CREATE TABLE t1;
 
 DROP TABLE t1;
+
```

Comment:
Do we need "End of 5.6 tests", "end of 5.7 tests" etc. markers in the testcase for easier future testcase maintenance WRT 5.7 -> 8.0 etc upmerges?

Reply:
Hmm, maybe not a bad idea if it helps the forward merging.